proxy/src/serverless/local_conn_pool.rs

```diff
@@ -475,20 +457,39 @@ impl LocalClient<tokio_postgres::Client> {
             )
             .await?;
 
-        info!(kid, jti = inner.jti, "user session state init");
+        info!(pid, jti = inner.jti, "user session state init");
 
         Ok(())
     }
 }
 
-fn sign_jwt(sk: &SigningKey, header: String, payload: String) -> String {
-    let header = Base64UrlUnpadded::encode_string(header.as_bytes());
-    let payload = Base64UrlUnpadded::encode_string(payload.as_bytes());
+fn sign_jwt(sk: &SigningKey, payload: &[u8]) -> String {
```

Comment:
I had a bit of time so I thought I'd try make this work with a single alloc only :)

Reply:
Yeah, probably should do that 😅